app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java

```diff
@@ -77,7 +77,7 @@
 
   @Override
   public void onCreate(Bundle savedInstanceState) {
-    getWindow().setWindowAnimations(R.style.WindowAnimationTransition);
+    //getWindow().setWindowAnimations(R.style.WindowAnimationTransition);
```

Comment:
If this works then instead of commenting this line you can delete this line.

Reply:
Okay.